ztp/source-crs/StoragePV.yaml

```diff
@@ -1,29 +0,0 @@
-apiVersion: v1
```

Comment:
This CR is part of the 4.13 release and could be used by customers for other PVs as well. I don't think it should be removed.